scripts/Dockerfile.devel

```diff
@@ -1,12 +1,18 @@
 FROM ceph/ceph:v15
 
 ENV GOPATH=/go \
-	GO111MODULE=on
+ GOROOT=/usr/local/go \
+ GO111MODULE=on
+
+RUN mkdir -p /usr/local/go && \
+ curl https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz | tar xzf - -C /usr/local/go --strip-components=1
+
+ENV PATH="$PATH:$GOROOT/bin:$GOPATH/bin"
```

Comment:
The order of this is incorrect. $PATH should be last, otherwise the golang executables may not get found when there is also an old version in the $PATH directories.

Reply:
@nixpanic we are not installing or base image dont have any go installation, so this should be fine.

Reply:
We do not know if an update to the base image gets an old version of golang installed. If that ever happens, we suddenly get very weird issues that are extremely difficult to understand.

Preventing that now, by setting the correct PATH _is_ important.

Reply:
@nixpanic sure, adjusted in latest patchset. Thought of avoiding one more CI run as we are at last minute..  any way.. adjusted and its working locally. PTAL.

```
cephcsi image settings: quay.io/cephcsi/cephcsi version canary
if [ ! -d ./vendor ]; then (go mod tidy && go mod vendor); fi
GOOS=linux GOARCH=amd64 go build -mod vendor -a -ldflags ' -X github.com/ceph/ceph-csi/pkg/util.GitCommit=d7cdbc0a714d46509c2a61c5a6785a020a7e6811 -X github.com/ceph/ceph-csi/pkg/util.DriverVersion=canary' -o _output/cephcsi ./cmd/
make: Leaving directory '/go/src/github.com/ceph/ceph-csi'
```